workshops/models.py

```diff
@@ -2054,6 +2054,18 @@ def clean(self):
     def get_absolute_url(self):
         return reverse('trainingrequest_details', args=[self.pk])
 
+    def __str__(self):
+        return (
+            '{state} training request by {personal} {family} <{email}>'
+            .format(
+                state=self.get_state_display(),
+                personal=self.personal,
+                family=self.family,
+                email=self.email,
+                timestamp=self.created_at,
+            )
+        )
+
```

Comment:
In search results, it would be better to see something like: 
```
Bob Smith <bob.smith@example.com>
```
or
```
Bob Smith <bob.smith@example.com> - pending
```
so that if you have i.e. 20 matches, it's easy to spot on the right one.